includes/base/widget-base.php

```diff
@@ -56,10 +129,30 @@ public function __construct( $data = [], $args = null ) {
 		}
 	}
 
+	/**
+	 * Show in panel.
+	 *
+	 * Whether to show the widget in the panel or not. By default returns true.
+	 *
+	 * @access public
+	 *
+	 * @return bool Whether to show the widget in the panel or not.
+	 */
 	public function show_in_panel() {
 		return true;
 	}
 
+	/**
+	 * Start widget controls section.
+	 *
+	 * Used to add a new section of controls to the widget. Regular controls and
+	 * skin controls.
```

Comment:
Registering controls - must be wrapped with `start_controls_section` and `end_controls_section`